auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -131,19 +126,19 @@ public void onCompleted(JSONObject object, GraphResponse response) {
                         FacebookRequestError requestError = response.getError();
                         if (requestError != null) {
                             Log.e(TAG, "Received Facebook error: " + requestError.getErrorMessage());
-                            mCallbackObject.onFailure(new Bundle());
+                            onFailure(new Bundle());
                             return;
                         }
                         if (object == null) {
                             Log.w(TAG, "Received null response from Facebook GraphRequest");
-                            mCallbackObject.onFailure(new Bundle());
+                            onFailure(new Bundle());
                         } else {
                             try {
                                 String email = object.getString("email");
-                                mCallbackObject.onSuccess(createIDPResponse(loginResult, email));
+                                FacebookProvider.this.onSuccess(email, loginResult);
```

Comment:
Any reason this branch has `FacebookProvider.this` but the failure branch does not?